plugins/inputs/leofs/leofs_test.go

```diff
@@ -131,7 +131,7 @@ func testMain(t *testing.T, code, endpoint string, serverType serverType) {
 	}
 
 	// Build the fake snmpwalk for test
-	src := os.TempDir() + "/test.go"
+	src := t.TempDir() + "/test.go"
```

Comment:
I really would feel better with `filepath.Join`... :-)

Reply:
Of course, changed.